src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40 * days;
+  const discount = 20;
+  const discountExtra = 50;
+
+  if (days < 3) {
+    return price;
+  } else if (days < 7) {
+    return price - discount;
+  }
```

Comment:
If you use return in each statement, else is redundant here
```suggestion
  if (days < 3) {
    return price;
  }
  
  if (days < 7) {
    return price - discount;
  }
```